package.json

```diff
@@ -109,7 +109,9 @@
   },
   "devDependencies": {
     "@jest/globals": "^27.3.1",
+    "@rauschma/stringio": "^1.4.0",
```

Comment:
hehe, Mr 2ality 😎

Reply:
Finally he's officially part of the project